pkg/cli/resources.go

```diff
@@ -197,8 +202,13 @@ func ResourcesProxy(rack sdk.Interface, c *stdcli.Context) error {
 	}
 
 	resourceUrl := r.Url
-	if !strings.HasPrefix(r.Url, "http") {
-		resourceUrl = fmt.Sprintf("http://%s", r.Url)
+	rex, err := regexp.Compile(RESOURCE_URL_PATTERN)
+	if err == nil {
+		// Test if the string matches the resource url pattern
+		if !rex.MatchString(resourceUrl) {
+			// add the scheme to the url
+			resourceUrl = fmt.Sprintf("https://%s", resourceUrl)
```

Comment:
previous one had `http`, but here we are adding `https://`?

Reply:
no reason, whether it's http or https the proxy won't use it, it's just used by the url parser.